woodwork/statistics_utils/_get_dependence_dict.py

```diff
@@ -103,24 +103,33 @@ def _get_dependence_dict(
     start_time = timer()
     if target_col is not None and target_col not in list(dataframe.columns):
         raise ValueError("target_col '{}' not in the dataframe".format(target_col))
-
+    boolean_columns = dataframe.ww.select(["Boolean", "BooleanNullable"]).columns.values
+    bool_to_int = {col: "IntegerNullable" for col in boolean_columns}
```

Comment:
Map boolean values to be `IntegerNullable` to allow them to be used in dependence calculations